ibazel/ibazel.go

```diff
@@ -449,6 +475,12 @@ func (i *IBazel) queryForSourceFiles(query string) ([]string, error) {
 		case blaze_query.Target_SOURCE_FILE:
 			label := *target.SourceFile.Name
 			if strings.HasPrefix(label, "@") {
+				parts := strings.Split(strings.TrimPrefix(label, "@"), "//")
```

Comment:
Could you break this right here into a helper function. I think it works in all cases but it would be nice to get a unit test on it. I'll even write it for you!

```
// Broken out func
func parseTarget(t string) (repo, target string) {
  parts := strings.Split(strings.TrimPrefix(label, "@"), "//")
  return parts[0]. parts[1]
}

// Test
func TestParseTarget(t *testing.T) {
  tests := []struct{
    in string
    repo string
    target string
  }{
    {"@//my:target", "", "my:target"},
    {"repo@//my:target", "repo", "my:target"},
    {"bazel_tools@//:strange/target", "bazel_tools", ":strange/target"},
  }
  for _, test := range tests {
    t.Run(test.in, func(t *testing.T) {
      gotRepo, gotTarget := parseTarget(test.in)
      if gotRepo != test.repo {
        t.Error("parseTarget(%q).repo = %q, want %q", test.in, gotRepo, test.repo)
      }
      if gotTarget != test.target {
       t.Error("parseTarget(%q).target = %q, want %q", test.in, gotTaret, test.target)
      }
    }
  }
}
}
```

Reply:
Sure